pkg/util/ipvs/kernelcheck_linux.go

```diff
@@ -47,7 +47,7 @@ func (r RequiredIPVSKernelModulesAvailableCheck) Check() (warnings, errors []err
 	// Find out loaded kernel modules
 	out, err := r.Executor.Command("cut", "-f1", "-d", " ", "/proc/modules").CombinedOutput()
 	if err != nil {
-		errors = append(errors, fmt.Errorf("error getting installed ipvs required kernel modules: %v(%s)", err, out))
+		errors = append(errors, fmt.Errorf("error getting installed ipvs required kernel modules: %v(%s)", err, string(out)))
 		return nil, errors
```

Comment:
should we change this?

Reply:
no, we shouldn't. reverted back. Thanks.